core/src/main/scala/org/apache/spark/sql/delta/Checkpoints.scala

```diff
@@ -290,18 +290,39 @@ object Checkpoints extends DeltaLogging {
     // log store and decide whether to use rename.
     val useRename = deltaLog.store.isPartialWriteVisible(deltaLog.logPath, hadoopConf)
 
-    val checkpointRowCount = spark.sparkContext.longAccumulator("checkpointRowCount")
-    val numOfFiles = spark.sparkContext.longAccumulator("numOfFiles")
```

Comment:
The reason we added these two accumulators is to verify the checkpoint we wrote is correct. We used to have a bug that we wrote an incomplete checkpoint, so we added these two accumulators for safety. If we compute them without writing the checkpointing, they would be meaningless. Could you keep them?

Reply:
Yeah I'll add them back since I'm getting rid of the agg now anyway

Reply:
Added back in